cfme/infrastructure/resource_pool.py

```diff
@@ -96,16 +92,39 @@ def _on_detail_page(self):
     @property
     def exists(self):
         try:
-            sel.force_navigate('infrastructure_resource_pool', context=self._get_context())
+            navigate_to(self, 'All')
             quad = Quadicon(self.name, 'resource_pool')
             if sel.is_displayed(quad):
                 return True
         except sel.NoSuchElementException:
             return False
 
 
+@navigator.register(ResourcePool, 'All')
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
+
+    def step(self):
+        from cfme.web_ui.menu import nav
+        nav._nav_to_fn('Compute', 'Infrastructure', 'Resource Pools')(None)
+
+    def resetter(self):
+        # Reset view and selection
+        tb.select("Grid View")
+        sel.check(paginator.check_all())
+        sel.uncheck(paginator.check_all())
+
+
+@navigator.register(ResourcePool, 'Details')
+class Details(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.click(Quadicon(self.obj.name, 'resource_pool'))
```

Comment:
Could set ResourcePool.quad_name attribute, and then reverence here like name.

I've seen both styles in-use in the code base, but this provides for the ability to update or version-pick that quad name once instead of every time you want to click it.